src/main/java/taxi/dao/CarDaoImpl.java

```diff
@@ -227,10 +227,12 @@ private Driver parseDriverFromResultSet(ResultSet resultSet) throws SQLException
         Long driverId = resultSet.getObject("id", Long.class);
         String name = resultSet.getNString("name");
         String licenseNumber = resultSet.getNString("license_number");
-        Driver driver = new Driver();
+        String login = resultSet.getNString("login");
+        String password = resultSet.getNString("password");
+        Driver driver = new Driver(name, licenseNumber);
```

Comment:
If you initialize an object with a constructor, do so for all fields, or use setters for all fields. Fix it everywhere.

Reply:
OK